Fabric.Authorization.UnitTests/EDWAdminSyncService/EDWAdminRoleSyncServiceTests.cs

```diff
@@ -1,14 +1,14 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using Fabric.Authorization.Domain.Models;
-using Fabric.Authorization.Domain.Services;
-using Fabric.Authorization.Domain.Stores;
-using Moq;
-using Xunit;
-
-namespace Fabric.Authorization.UnitTests.EDWAdminSyncService
+﻿namespace Fabric.Authorization.UnitTests.EDWAdminSyncService
 {
+    using System;
```

Comment:
while I would normal suggest this change, if you look at the rest of the code in this solution, you will see the style does not support this change.

To keep consistent, i would change it back.